etna/pipeline/hierarchical_pipeline.py

```diff
@@ -73,8 +88,12 @@ def raw_forecast(
             Dataset with predictions at the source level
         """
         forecast = super().forecast(prediction_interval=prediction_interval, quantiles=quantiles, n_folds=n_folds)
+        column_names = forecast.columns.get_level_values(level=1)
+        target_columns = match_target_quantiles(column_names)
+        target_columns.add("target")
```

Comment:
In some places you create tuple, and here you work with set, may you make it consistent in all the places